modules/claims_api/app/swagger/claims_api/v1/swagger.json

```diff
@@ -8944,14 +8962,32 @@
                                 "description": "Service Branch for the veteran.",
                                 "type": "string",
                                 "enum": [
-                                  "AIR FORCE",
-                                  "ARMY",
-                                  "COAST GUARD",
-                                  "MARINE CORPS",
-                                  "NAVY",
-                                  "OTHER"
```

Comment:
Unfortunately, I don't think we can remove these uppercased values.
Since this is a released API, our consumers have built logic that uses those uppercased values and if we take the uppercased values away, JSON schema validation will begin failing for any consumer sending us those uppercased values.